### DISCUSSION TITLE
Java 11 is the new minimum for iCal4j v4

### DISCUSSION THREAD

**Discussion by benfortuna** [ical4j/ical4j#679] [General]
Prior to iCal4j v4.0.0-rc4 we were building iCal4j libraries with Java 8 compatibility. A number of factors lead to decision to to move to a minimum Java 11, including:

* Support for Java module system (since Java 9)
* Java 11 is the next LTS release after Java 8

iCal4j v3.x will continue to support Java 8, however will move to maintenance mode after v4 final is released.

**Comment by ChMThiel** [comment]
@benfortuna When will a 4.0.0 Release be available? There have been quite a lot of RCs...we require a released lib!

**Reply by benfortuna** [reply]
I think we can do a 4.0 release if that helps. I'll make the next one a GA (4.0.0) release.

**Reply by benfortuna** [reply]
http://ical4j.github.io/2024/06/19/ical4j-v4-released.html
